src/mcif/__init__.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-__version__ = '1.0'
-__benchmark_version__ = '1.0'
+__version__ = '1.1'
```

Comment:
```suggestion
__version__ = '1.0'
```
we have not changed the evaluation code and we have not yet released it on pypi, so I see no need to upgrade the version